evil-types.el

```diff
@@ -370,6 +370,45 @@ If visual state is inactive then those values are nil."
   (when (evil-ex-p)
     (evil-ex-get-substitute-info evil-ex-argument t)))
 
+(evil-define-interactive-code "<d/>"
+  "Ex delete argument."
+  (when (evil-ex-p)
+    (evil-ex-get-delete-info evil-ex-argument)))
+
+(defun evil-ex-get-delete-info (string)
+  "Parse STRING as a :delete argument.
+Return a list (REGISTER COUNT)"
+  (let* (
+         (split-args (split-string (or string "")))
+         (arg-count (length split-args))
+         (arg0 (nth 0 split-args))
```

Comment:
The general convention is to use `car` for the first element, `cadr` for the second and `nth` for anything with a greater index than 1.